delfin/api/v1/alerts.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
plz rename current alert.py to alert_source.py

Reply:
Updated